src/components/slop-fest-modal.jsx

```diff
@@ -19,7 +19,7 @@ export function SlopFestModal({ buttonTitle, location, fest }) {
   const festEnd = new Date(fest?.data?.fest?.endDate);
 
   useEffect(() => {
-    if (location.pathname === `/fests/${festId}`) {
+    if (location && location.pathname === `/fests/${festId}`) {
```

Comment:
nit: In places like this, you can use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining) which makes it look a bit cleaner and has the same functionality. You can see it in action right under your change below, on line 82.